backend/segmentation/tests/unit/test_preprocessing_pipeline.py

```
@@ -79,9 +79,13 @@ class TestPreprocessRGBImage:
     """_load_and_preprocess_image returns the correct tensor shape."""
 
     def test_preprocess_rgb_image(self, inference_service):
-        """RGB JPEG bytes → 4-D tensor (1, 3, H, W) and original size tuple."""
+        """RGB JPEG bytes → 4-D tensor (1, 3, H, W), original size tuple, and gray ndarray.
+
+        _load_and_preprocess_image() returns a 3-tuple:
+          (image_tensor, original_size, original_gray)
+        """
         img_bytes = _make_image_bytes(320, 240, mode="RGB", fmt="JPEG")
-        tensor, orig_size = inference_service._load_and_preprocess_image(img_bytes)
+        tensor, orig_size, orig_gray = inference_service._load_and_preprocess_image(img_bytes)
```

Ruff (0.15.14)

[warning] Unpacked variable `orig_gray` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)